### ISSUE TITLE
Implement softmax+sampler

### ISSUE THREAD

**Issue by qualiteger** [qualiteg/ChatStream#45]
logits にSoftmax 処理を行い、最終的に1件のトークンをサンプリングするSoftmaxSamplerを実装する

#42 のサブタスク

**Comment by qualiteger** [comment]
#42 のサブタスク
